client/src/CodeClaimPage/index.tsx

```diff
@@ -46,66 +46,75 @@ export const CodeClaimPage: React.FC<RouteComponentProps<{ hash: string, method:
   }, []); /* eslint-disable-line react-hooks/exhaustive-deps */
 
   useEffect(() => {
-    if (claim && claim.delegated_mint && !isVerified) {
-      verifySignedMessage()
+    if (claim && claim.email_claimed) {
```

Comment:
I saw in this line https://github.com/Xivis/poap/pull/62/files#diff-e495969206e2f262ad89b8398a238fdfR20 that we use an optional operator, why not here? We should begin using it more consciously why are we using vs instead using it because it's free and avoid us possible errors since it changes the meaning of the code.

Reply:
You are right. I should have use it the same way in both places. In this case I think is better not to use the conditional operator because we know it's a field that will always come, either true or false, but will never be non existent. 

Are you saying that we should it every time we use a dot notation (i.e. `object?.property1?.property2)